corehq/apps/reports/tasks.py

```diff
@@ -370,7 +370,7 @@ def _get_export_properties(export):
     return properties
 
 
-def find_question_id(form, value):
+def find_question_id(form, value, last_path=False):
```

Comment:
What do you think of renaming this parameter to `use_basename`? It's meaning was not immediately obvious to me, but that might be my problem not yours.